client/src/containers/CampaignAdmin/tests/CampaignAdminItem-test.js

```diff
@@ -0,0 +1,149 @@
+/* global jest, describe, beforeEach, it, expect */
+
+jest.unmock('react');
+jest.unmock('react-addons-test-utils');
+jest.unmock('../CampaignAdminItem');
+
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import CampaignAdminItem from '../CampaignAdminItem';
+
+describe('CampaignAdminItem', () => {
+  let props = null;
+
+  beforeEach(() => {
+    props = {
+      campaign: {},
+      item: { ID: 1 },
+    };
+  });
+
+  describe('Links', () => {
+    let cmp = null;
+    let tooltip = null;
+
+    beforeEach(() => {
+      cmp = ReactTestUtils.renderIntoDocument(
+        <CampaignAdminItem {...props} />
+      );
+      tooltip = ReactTestUtils
+        .scryRenderedDOMComponentsWithClass(cmp, 'campaign-admin__item-links');
+    });
+
+    it('should not show link icon by default', () => {
+      expect(tooltip).toHaveLength(0);
+    });
+
+    it('should show link icon when the item is being from the selected item', () => {
+      props.isLinked = true;
+      cmp = ReactTestUtils.renderIntoDocument(
+        <CampaignAdminItem {...props} />
+      );
+      tooltip = ReactTestUtils
+        .scryRenderedDOMComponentsWithClass(cmp, 'campaign-admin__item-links');
+
+      expect(tooltip).toHaveLength(1);
+      expect(tooltip[0]).toBeInstanceOf(HTMLElement);
```

Comment:
Actually, this line is just redundant to the one above it. I'll just remove it. The purpose of this test case is to make sure the link is element is there. The tooltip text and and number are tested in the cases below.